### DISCUSSION TITLE
CVE-2024-47220 WEBrick < 1.8.2

### DISCUSSION THREAD

**Discussion by NathanG8** [rubycdp/ferrum#497] [General]
Is there any plans to update `ferrum` so it is no longer restricted to `1.7 `and can use `1.8.2` specifically to resolve the CVE?

NVD CVE: https://nvd.nist.gov/vuln/detail/CVE-2024-47220

**Comment by route** [comment]
It's development dependency, should be ok. But PRs are appreciated

**Reply by guigs** [reply]
@route Should it be a `add_development_dependency` then? I would like to avoid installing `webrick` as dependency when I add `ferrum` to my project if it is not needed. But I see it is required in `Ferrum::Proxy`.

**Reply by route** [reply]
That's right I was wrong it's not a dev dependency. It is used for proxies. So I'm afraid it's a requirement now
